buildSrc/subprojects/performance/src/main/groovy/org/gradle/testing/ReportGenerationPerformanceTest.groovy

```diff
@@ -39,30 +40,30 @@ abstract class ReportGenerationPerformanceTest extends PerformanceTest {
     @PathSensitive(PathSensitivity.RELATIVE)
     File reportDir
 
+    @OutputFile
+    @PathSensitive(PathSensitivity.RELATIVE)
+    File resultsJson
```

Comment:
Is this an output file and an input file? Do we want to share this file via the build cache?

If we don't want to share the file via the build cache, maybe `@LocalState` would be better suited. Or maybe the task is only cacheable if there is no `resultsJson`, given that it influences the output, right?

Can we declare this file an output for the initial run and an input for the re-run?